### ISSUE TITLE
A funny problem while testing

### ISSUE THREAD

**Issue by MahyarFardin** [Elman295/TransUnet-for-Surgical-Scene-Segmentation#1]
Hi, 
I am not a pytorch expert but I believe that you have to write 
`
with torch.no_grad():
    model.eval()

`
before evaluating your model with either train or test datasets. Otherwise dropout and batchnorm layers would yet effect the model.

**Comment by Elman295** [comment]
Hello, 
Thanks for your clear comment. yes, you are right.
I will correct my code soon. 

Best regards, Elman
